solver/app/quote_internal_test.go

```diff
@@ -265,7 +265,7 @@ func TestQuote(t *testing.T) {
 			res: types.QuoteResponse{
 				Deposit: types.AddrAmt{
 					Token:  erc20(evmchain.IDBase, tokens.USDC).Address,
-					Amount: parseInt("5015000000000000000"), // Price is $5/OMNI
```

Comment:
this test was incorrect, it swapped 501500000000 USDC for 1 OMNI.

Now it swaps 5.015USDC for 1 OMNI